File: C#/leap/Leap.cs
using System;

public static class Leap
{
    public static bool IsLeapYear(int year)
    {
        bool divBy4 = year % 4 == 0;
        bool divBy100 = year % 100 == 0;
        bool divBy400 = year % 400 == 0;
        
        return (divBy4 && !divBy100 && !divBy400) || (divBy4 && divBy400);
    }
}

File: JavaScript/leap.js
//
// This is only a SKELETON file for the 'Leap' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const isLeap = (year) => {
  let divBy4 = year % 4 == 0;
  let divBy100 = year % 100 == 0;
  let divBy400 = year % 400 == 0;

  return (divBy4 && !divBy100 && !divBy400) || (divBy4 && divBy400);
};
